src/acquisition/covid_hosp/state_daily/network.py

```diff
@@ -14,3 +20,39 @@ def fetch_metadata(*args, **kwags):
 
     return Network.fetch_metadata_for_dataset(
         *args, **kwags, dataset_id=Network.DATASET_ID)
+
+  def fetch_revisions(newer_than):
+    """Scrape CSV urls from the HHS revisions pages.
+
+    The revisions page is a big table showing recent revisions to the state
+    daily dataset. Each row represents a single revision with a link to a
+    separate page where a download is available for that revision. 
+
+    The first row is different, and links directly back to the main dataset
+    page. Since this row represents the current published version, and the URL
+    for that CSV is included in the metadata, we do not retrieve it here.
+
+    The link for each revision is a date in the same format as
+    `revision_timestamp` from the metadata, so we bookend which links we retrieve
+    by checking against the issue code (YYYYMMDD) for the last issue we
+    imported. 
+
+    This approach will break under the following conditions:
```

Comment:
I assume we're ok with these given they're documented here? If we intend on fixing in the future, should create a github issue for it.

Reply:
I am grudgingly okay with it? a robust solution would switch the metadata "have we seen this update before" approach to track the CSV filename instead of revision_timestamp, but that seemed like way more work + greater opportunity to screw up the other covid_hosp pipelines.

We may want to review this with Roni and/or Nick before merging.